04_numeronym/kasunfdo/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(s ...string) []string {
+	out := make([]string, len(s))
+	for i, str := range s {
+		out[i] += getNumeronym(str)
+	}
+	return out
+}
+
+func getNumeronym(str string) string {
```

Comment:
Naming a function "getSomething" usually implies a "getter" function. In Go, these are named without the Get prefix (https://golang.org/doc/effective_go.html#Getters). However this isn't even a getter, so it makes sense to not use the Get prefix.

Since you have `numeronyms` as the name of the function generating multiple numeronyms, I think a more sensible name for this function that is more in line with idiomatic Go would be just `numeronym`. That makes `numeronyms` produce multiple numeronyms and `numeronym` produce one numeronym. It is a more consistent naming.